tpls/desul/include/desul/atomics/Thread_Fence_OpenMP.hpp

```diff
@@ -16,7 +16,7 @@ SPDX-License-Identifier: (BSD-3-Clause)
 namespace desul {
 namespace Impl {
 
-#if _OPENMP > 201800
+#if _OPENMP > 201800 && !defined(KOKKOS_COMPILER_NVHPC)
```

Comment:
FIXME?

Reply:
Added a fixme already. Should have done that in the first commit itself.